ceph-salt-formula/salt/_states/ceph_orch.py

```diff
@@ -29,12 +29,14 @@ def set_admin_host(name, timeout=1800):
             if provisioned:
                 ssh_user = __pillar__['ceph-salt']['ssh']['user']
                 sudo = 'sudo ' if ssh_user != 'root' else ''
+                home = '/home/{}'.format(ssh_user) if ssh_user != 'root' else '/root'                
                 status_ret = __salt__['cmd.run_all']("ssh -o StrictHostKeyChecking=no "
-                                                     "-i /tmp/ceph-salt-ssh-id_rsa {}@{} "
+                                                     "-i {}/.ssh/id_rsa {}@{} "
```

Comment:
can we just drop `-i {}/.ssh/id_rsa` altogether, since it is the default?

Reply:
I think we cant, see:
```
master:~ # ssh -o StrictHostKeyChecking=no -i /home/ceph-salt/.ssh/ ceph-salt@master.pacific.test ls
authorized_keys  id_rsa           id_rsa.pub      

master:~ # ssh -o StrictHostKeyChecking=no -i /home/ceph-salt/.ssh/id_rsa ceph-salt@master.pacific.test ls
bin
```